Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -192,6 +193,143 @@ void JitArm64::DoDownCount()
   gpr.Unlock(WA, WB);
 }
 
+void JitArm64::ResetStack()
+{
+  LDR(INDEX_UNSIGNED, X0, PPC_REG, PPCSTATE_OFF(stored_stack_pointer));
+  SUB(SP, X0, 16);
+}
+
+void JitArm64::WriteLKExit(u32 destination, u32 after)
```

Comment:
`after` should probably be `exit_address_after_return` or something more indicative to what it is. Ditto for the two other functions below that also have `u32 after`